### DISCUSSION TITLE
Add "general bandos" as an alternative for General Graardor

### DISCUSSION THREAD

**Discussion by expired-data** [runelite/runelite#15316] [Ideas]
In the kc quick chat plugin it calls general bandos something like general graardor, but this isn't right! His name is general bandos I'm pretty sure. We could fix this by making it says general bandos and supporting `!kc general bandos` as a chat command

**Comment by michmest** [comment]
Have been noticing this as well! Would make sense to me to make it consistent with the format !kc [boss name]. +1 for change!

**Reply by Enriath** [reply]
The boss's name is General Graardor. The command is correct at the moment. Adding an alternative for this incorrect name would be fine, and is something we've done for multiple others.

**Reply by expired-data** [reply]
Are you sure the boss is called General Graardor? I only got in the room long enough to use my dragon dagger on him and did remember to say sit afterwards, but I'm fairly certain that it could be called General Bandos, since that's why everyone just shortens him to bandos

**Reply by michmest** [reply]
Oh apologies I must have been misinformed. Me and my friends always call him that way! I'll make sure to spread the message.
